Add flavor information

mfmsilva opened the pull request:
Added required flavors for juju controller, k8s master and worker.

mfmsilva committed:
Add flavor information

Added required flavors for juju controller, k8s master and worker.

majduk commented:
Please prepare a list of flavors with extra specs for:
- LMA nodes (juju controller et consortes)
- control nodes (kubernetes worker and collocated services)
- k8s worker nodes.

Last part is where we are missing verbosity. k8s worker nodes can come in two flavors. Basic and Network intensive. This set of flavors has only configuration for basic worker type. Intensive one should have a description of hugepages, numa requirements, cpu allocation, etc.

mfmsilva committed:
Flavor and Kubelet plugins

Changed flavor information to contain basic and intensive profiles.
Added mandatory Kubelet plugins

mfmsilva committed:
Fix syntax
